### PR TITLE
Fixed Bot-Task-Module Sample Issues

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

File: samples/bot-task-module/nodejs/pages/TaskModuleTab.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
     var taskModuleButtons = document.getElementsByClassName("taskModuleButton");
     if (taskModuleButtons.length > 0) {
+        fetch(`${window.location.origin}/AuthTab`).then(response => response.json()).then(data => {
 
         //config your appid and baseurl
         var config = {
-            MicrosoftAppID: "<<Microsoft-App-ID>>",
-            BaseUrl: "<<BaseUrl>>"
+            MicrosoftAppID: data.MicrosoftAppId,
+            BaseUrl: `${window.location.origin}`
```
Comment (ChetanSharma-msft) [inline samples/bot-task-module/nodejs/pages/TaskModuleTab.js:12]: Will it work in Desktop and mobile clients?  ${window.location.origin}

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added one review comment.

File: samples/bot-task-module/nodejs/pages/TaskModuleTab.js (hunk 1 above)
Comment (ChetanSharma-msft) [reply samples/bot-task-module/nodejs/pages/TaskModuleTab.js:12]: @Harikrishnan-MSFT - Please run build pipeline as well.

Comment (Harikrishnan-MSFT) [reply samples/bot-task-module/nodejs/pages/TaskModuleTab.js:12]: It is working in Desktop clients.

Comment (Harikrishnan-MSFT) [reply samples/bot-task-module/nodejs/pages/TaskModuleTab.js:12]: ****Build** Pipeline -** 
image

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5423570458

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Prithvi-MSFT — APPROVED** [review]

**Pull request merged**